ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -281,6 +281,9 @@ def database_row_to_elasticsearch_doc(row, schema):
             extension=Audio.get_extension(row[schema['url']]),
             categories=row[schema['category']],
             license_url=Audio.get_license_url(meta),
+            bit_rate=row[schema['bit_rate']],
+            sample_rate=row[schema['sample_rate']],
+            genre=row[schema['genre']],  # Not sure how to map array
```

Comment:
Based on [this discussion](https://github.com/elastic/elasticsearch-dsl-py/issues/95#issuecomment-487974464), I think just setting an array as the value works.